### PR TITLE
Simple Time Measurements

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Simple log message for making it easier to identify slow test parts.

**Related issue(s)**

### CODE REVIEW

File: internal/broker/poller.go [hunk 1]
```diff
@@ -43,3 +43,18 @@ func (p *PassthroughPoller) Invoke(logic func() (bool, error)) error {
...
+
+type TimerPoller struct {
+	PollInterval time.Duration
+	PollTimeout  time.Duration
+	Log func(args ...any)
+}
+
+func (p *TimerPoller) Invoke(logic func() (bool, error)) error {
+	var start = time.Now()
+	result := wait.PollImmediate(p.PollInterval, p.PollTimeout, func() (bool, error) {
```
Comment (jaroslaw-pieszka) [inline internal/broker/poller.go:55]: `pollImmediate` is deprecated in latest Go version.

File: internal/broker/poller.go [hunk 2]
```diff
@@ -43,3 +43,18 @@ func (p *PassthroughPoller) Invoke(logic func() (bool, error)) error {
...
+		return logic()
+	})
+	p.Log(fmt.Sprintf("Waiting for the status took: %v", time.Since(start)))
```
Comment (jaroslaw-pieszka) [inline internal/broker/poller.go:58]: It is not easy to comprehend that we call `logic` to get `status`.  Either comment or more consistent names would be beneficial.

File: internal/broker/poller.go (hunk 1 above)
Comment (ralikio) [reply internal/broker/poller.go:55]: We decided not to do it right now.

File: internal/broker/poller.go (hunk 2 above)
Comment (ralikio) [reply internal/broker/poller.go]: Changed the message.

**Pull request merged**